### ISSUE TITLE
An example of Boolean hyperparameter for adding BatchNormalization()

### ISSUE THREAD

**Issue by rahulroynit** [keras-team/keras-tuner#330]
Can you share an example that can help to understand the use of hp.Boolean for adding or leaving BatchNormalization()? Thanks!

**Comment by yixingfu** [comment]
The example for conditional hyperparameter (https://keras-team.github.io/keras-tuner/#the-search-space-may-contain-conditional-hyperparameters) is quite similar to what you wanted. Just replace the for-loop with if-statement.

**Comment by rahulroynit** [comment]
If I am using a conditional if-statement, I assume I don't need hp.Boolean() as I can accomplish it using some conditional variable in the if-statement. How hp.Boolean() is making it easier? Correct me if I am missing something.
